### DISCUSSION TITLE
Change License of this project

### DISCUSSION THREAD

**Discussion by wh201906** [wh201906/SerialTest#26] [General]
@Ferenc- @Poussinou @taotieren 
Hi. I'd like to change the license of this project from LGPL-3.0 to GPL-3.0. Would you allow me to do this?
I want to change the license because the dependency QCustomPlot uses GPL-3.0-or-later license, I think this license is not compatible to the LGPL. To avoid the related legal issues, I need to make this conversion. I hope you can understand my decision. Thank you.
***
大家好。我想将这个项目的许可证从LGPL-3.0更改为GPL-3.0。你们允许我这样做吗？
这个项目的依赖库 QCustomPlot 使用的是GPL-3.0-or-later许可证，作为调用方我应该不能使用LGPL协议。为避免相关的法律问题，我需要把项目的协议变为GPL。希望你们能理解我的决定。谢谢。

**Comment by Ferenc-** [comment]
You have my full consent.

**Reply by wh201906** [reply]
Thanks!

**Comment by Poussinou** [comment]
I only added the F-Droid button in the README, but you have my full consent to switch to GPL 3 anyway with everything I did in this repo 👍

**Reply by wh201906** [reply]
Thanks!

**Comment by taotieren** [comment]
同意变更。

**Reply by wh201906** [reply]
感谢！
